styles.css

```diff
@@ -552,14 +547,14 @@ input::-webkit-input-placeholder {
   width: 100px;
   display: block;
   color: #fff;
-  transform: translate(14px, 40px);
+  margin: 42px 0 0 12px;
```

Comment:
- no need left margin
- top margin should be set as a padding on the parent element (`.nav-block`)